src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -20,24 +21,28 @@
 
 class UnaryUnaryMultiCallable(aio.UnaryUnaryMultiCallable):
 
-    def __init__(self, channel, method, request_serializer,
-                 response_deserializer):
+    def __init__(self, channel: cygrpc.AioChannel, method, request_serializer,
+                 response_deserializer) -> None:
```

Comment:
Since you just added type to `channel`, why not add types to all arguments? It would probably be simpler to predefine those types, since they might be reused for other `Callable` class.